test/linalg/dense.jl

```diff
@@ -512,6 +512,20 @@ end
     @test_throws ArgumentError diag(zeros(0,1),2)
 end
 
+@testset "Matrix to real power" for elty in (Float64, Complex{Float64})
+# Tests proposed at Higham, Deadman: Testing Matrix Function Algorithms Using Identities, March 2014
+    A11 = rand(elty, 10, 10)
```

Comment:
I would rather do a fixed `A11 = [....]` matrix here, though, than a random one.  We don't want the tests to fail just because `rand` was unlucky.  Also, we should probably test a matrix that is close to defective, which is unlikely with `rand`.